maven/src/main/java/cz/o2/proxima/maven/ModelGenerator.java

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright 2017-2018 O2 Czech Republic, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.o2.proxima.maven;
```

Comment:
This class looks unrelated to maven, can we use a different package? i.e. `cz.o2.proxima.generator`?

Reply:
Sounds good. Will do.